cypress/integration/tests/signIn/signIn.js

```diff
@@ -2,11 +2,57 @@ import SignIn from '../../../page-objects/signInPage';
 import credentials from '../../../fixtures/credentials.json';
 
 describe('Sign In', () => {
-  it('TA-1: User is able to log in with valid credentials', () => {
-    SignIn.navigateToSignInPage();
-    SignIn.enterSignInEmail(credentials.email);
-    SignIn.enterSignInPassword(credentials.password);
+
+  it('TA-1: User is able to sign in with valid credentials', () => {
+    SignIn.open('/signing');
+    SignIn.enterSignInEmail(credentials.emailValid);
+    SignIn.enterSignInPassword(credentials.passwordValid);
+    SignIn.submitSignInForm();
+    SignIn.signOutIcon.should('be.visible');
+  });
+
+  it('TA-2: User unable to sign in with a wrong password', () => {
+    SignIn.open('/signing');
+    SignIn.enterSignInEmail(credentials.emailValid);
+    SignIn.enterSignInPassword(credentials.passwordInvalid);
+    SignIn.submitSignInForm();
+    SignIn.signInError.should('contain.text', 'The password is invalid or the user does not have a password.');
```

Comment:
create fixture file for errors / messages and import them to your tests
it would be better to handle all hardcoded value in one file

Reply:
done